infra/main.tf

```diff
@@ -0,0 +1,107 @@
+resource "azurerm_resource_group" "nachet-resources-group" {
+  name     = "nachet"
+  location = "canadacentral"
+}
+
+resource "azurerm_container_registry" "nachet-registry" {
```

Comment:
I'm guessing we'll make this common infrastructure ? I'm imaging that one artifact registry per project is overkill

Reply:
Ok, we can do that.

Reply:
either that or a new repo infra (don't forget we'll probably have to fork those IaC repos back to our official subscriptions).